examples/components/build-test-components.sh

```diff
@@ -0,0 +1,2 @@
+node output/export-components.js examples/components/ex-progressbar.html examples/components/ex-temperature.html examples/components/ex-container.html examples/components/ex-colorpicker.html -o examples/components/example-components.js
+node output/export-components.js examples/components/ex-colorpicker.html -o examples/components/ex-colorpicker.js
```

Comment:
Right now, this is just a shell script, but we should migrate this with a JS API (separate issue https://github.com/Tram-One/tram-lite/issues/41)